### PR TITLE
Omni-platform changes, Sprites

### PR DESCRIPTION
**Pull request by SouthyOsmanli**
## About The Pull Request
<details>
<summary>
	Changed the Omnirifle platform guns. The Hustler has lost its bonus damage, the Warthog has gained the bonus damage (as the Hustler is meant to be 'shit' compared to the other Omnis by description) and the Boar has been made slightly more usable. Also - lowered Blackguard attachment slots by 1 and made the SWAT shotgun be able to compete with the WO's osprey. Similar done to the Ostwind to make it competitive with other armory guns.
</summary>
<hr>
	
<hr>
</details>

## Changelog
:cl:
balance: Rebalanced Omni platforms (Hustler, Omni, Boar, Warthog). Mostly the stats on the Boar being slightly better, the Hustler losing its bonus damage/pen, Warthog gaining damage/pen since it is rarer.
balance: Blackguard moved from 4 attachment slots to 3. Stops it from being 'gamer gun'.
balance: SWAT Shotgun buffed slightly to be able to compete with the Osprey as a primary firearm.
balance: Slight buff to the Ostwind to get rid of its negative-damage modifier and actually encourage it being used as a scoped weapon.
/:cl:

### CODE REVIEW

**Comment by Trilbyspaceclone** [comment]
This is a bit to controversial

**Comment by SouthyOsmanli** [comment]
They boo me because I am right...........

**Comment by Churro55** [comment]
Lowering mags to 12 is a bit too much. For the comparative damage difference between 7.5 and 408, especially with the added 257 and 7.5 Drum mag additions and this nerf, why would you ever use a 408 now? They have more recoil too.
The Blackguard is also limited to only two in the armory.

**Comment by SouthyOsmanli** [comment]
> Lowering mags to 12 and drums to 24 is a bit too much. For the comparative damage difference between 7.5 and 408, especially with the added 257 and 7.5 Drum mag additions and this nerf, why would you ever use a 408 now? They have more recoil too. The Blackguard is also limited to only two in the armory.

After talking to HOUND, I was told the Blackguard was not supposed to take **any** attachments in exchange for it's damage and pen. It also takes a drum, while the Omnirifle Longarm cannot. It's objectively one of the best guns in the armory. This is a balance to it to try to encourage other weapons be used as DMRs instead of the Blackguard being a full tradeoff and superior in every way.

The .408 changes are due to the fact .408 are currently meta. They have high damage, high pen and near equal recoil to some of their counterparts that were designed to compete in the same function. Instead of nerfing .408 damage and/or pen, I've kept them as it but simply made them limited-in-use by decreasing capacity. Most can take drums now, not only the Blackguard, so this only hurts DMRs the most. An Omni can still get it's normal 20 round mag (now 24), so this is not a major change to its ammo capacity.

**Comment by Churro55** [comment]
> 
> After talking to HOUND, I was told the Blackguard was not supposed to take **any** attachments in exchange for it's damage and pen. It also takes a drum, while the Omnirifle Longarm cannot. It's objectively one of the best guns in the armory. This is a balance to it to try to encourage other weapons be used as DMRs instead of the Blackguard being a full tradeoff and superior in every way.
> 
> The .408 changes are due to the fact .408 are currently meta. They have high damage, high pen and near equal recoil to some of their counterparts that were designed to compete in the same function. Instead of nerfing .408 damage and/or pen, I've kept them as it but simply made them limited-in-use by decreasing capacity. Most can take drums now, not only the Blackguard, so this only hurts DMRs the most. An Omni can still get it's normal 20 round mag (now 24), so this is not a major change to its ammo capacity. You just now require a drum, which are mildly harder to get (not really) to get the old ~20 round capacity instead of base-gun. Otherwise why would you ever use a 7.5 firearm at all when its normal printable mags, besides Guild or exotic disk find, is 10 or 20 round mags.

I'm guessing the Blackguard is supposed to be an straight upgrade, seeing as how there's only two and locked behind the Armory, though I think it's stupid that it can use a drum.
In general, MOST .408 shouldn't be able to use drums altogether, where it's use should be relegated to Heavy Weapons or specialized/unique weapons only.
Instead of capping the mag sizes further or it's damage/pen values, I suggest increasing it's recoil; it makes sense and it's the .408 niche anyhow.
.408 Drum mags are also only used in events for the most part and people don't ask the Guild for them (They don't print them ever anyways). 
Now with the 50 round 7.5 mags, why would you ever use a .408? Yes, they hit a bit harder, but with the more than double capacity and auto-fire I think it's objectively better if this change goes through.
The difference, at least in PvE, is not very noticeable. you'll easily run out of .408 quick and the boxes contain the least amount of rounds (makes sense) so you don't have many reloads at hand.

Tl;Dr: capacity and damage/pen reduction bad. Touch recoil and drum compatability instead.

**Comment by Kazkin** [comment]
Omni rifles are supposed to represent the pinnacle of battle rifle designs, with their only true downside being higher recoil. Omni is to 7.5 what .50 kurtz is to .40/.35, it is meant to be a direct upgrade by comparison except for its harder to obtain rifles and sometimes munitions. Omni rifles are meant to be high end dungeon rewards (such as the greyson gauntlet) or standard issue for the blackshield/marshals as the game stopper for other enemies who might come with good quality weapons. I can't say I agree with this PR as it goes against the fundamental and original design choices made by both me and Heedless, the original spriter/designer for these weapons.

As an aside, the insane recoil was meant to be purposeful as lore wise, omni rifles are standard issue for terran marines who use power armor to give them no recoil at all, making the weapon alot less effective if stolen/taken/used by anyone but a terran federation soldier.

**Comment by cdb-is-not-good** [comment]
I do think this wouldn't completely nullify the advantages of .408s, it'd simply make their benefits less extreme versus other options. Afterall, we did get rid of the insane recoil, fluff/mechanics aside that provided quite a large boon to these weapons and pushed them further into 'master of all' category

**Comment by SouthyOsmanli** [comment]
> 

I haven't done anything major besides reduce the capacity on the drums, since all automatics and a few semi-auto .408s have gotten the ability to take it.

The only 'nerf' has been given to the Hustler, the Boar and Warthog got buffed. Omni, I think, I gave slightly higher recoil. 

Also - the choices aren't holding up because the Hustler has the best stats despite description wise being described 'as shit' (it's better than the Warthog by far, the Boar by far, and I'd say only beaten really by the Omni's automatic capability and the Longarm; which is just a shittier Blackguard now) - and the fact their prices are VERY, very low. Lower than an STS-30.

**Comment by SouthyOsmanli** [comment]
Do note: These magazine changes aren't even that strong of a 'nerf' since the drum holds 24 rounds. This is effectively pre-drum mag implementation. This only really nerfs the Blackguard, and the Fenrir (But - if you used a drum on the Fenrir anyway, I don't know why the hell you would. Boxes fit better.) 

.408s are objectively superior in every way to 7.5s. This is to attempt to address that. I was **asked** not to touch recoil much in terms of balance due to Trilby looking at a recoil rework PR.

**Comment by Churro55** [comment]
The Longarm is supposed to be the CO's unique gun. Why is it being nerfed?

Comparing .408's sharpshoot ability to lower calibers doesn't make sense. They are, and should be better than all of them, always.
Automatic fire however, not so much.
A bit busy so can't explain more but in a nutshell ^

**Comment by SouthyOsmanli** [comment]
> The Longarm is supposed to be the CO's unique gun. Why is it being nerfed?
> 
> Comparing .408's sharpshoot ability to lower calibers doesn't make sense. They are, and should be better than all of them, always. Automatic fire however, not so much. A bit busy so can't explain more but in a nutshell ^

Did you actually look at the changes of the PR? It's not. It's just worse than the Blackguard - objectively. Only change I did was increase it's price - because - yeah, it's worth way more than what was listed.

**Comment by SouthyOsmanli** [comment]
People were to dent-brained to explain how this effectively was just returning .408 magazines back to their previous pre-recent-buff stage so I removed the .408 capacity nerf orginally here.

Instead I've simply reverted the ability for Omni's to take drum mags instead.

File: code/modules/projectiles/ammunition/magazines.dm [hunk 1]
```diff
@@ -605,7 +605,7 @@
 /obj/item/ammo_magazine/heavy_rifle_408
 	name = "heavy rifle magazine"
 	icon_state = "hrifle"
-	desc = "A 20 round magazine marked for .408 Omni."
+	desc = "A 12 round magazine marked for .408 Omni."
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/ammunition/magazines.dm:608]:
```suggestion
	desc = "A 20 round magazine marked for .408 Omni."
```
